src/Attachment/File.php

```diff
@@ -52,14 +57,16 @@ class File
      *
      * @param UploadedFile $file
      * @param string|null  $disk
+     * @param string|null  path
      * @param string|null  $group
      */
-    public function __construct(UploadedFile $file, string $disk = null, string $group = null)
+    public function __construct(UploadedFile $file, string $disk = null, string $path = null, string $group = null)
```

Comment:
Warning: This is a breaking change and therefore cannot be released under the 10.x branch